src/AbstractFuzzyQuery.php

```diff
@@ -68,7 +75,12 @@ public function __toString()
             $raw = $this->exclusion . $raw;
         } else {
             if (!empty($this->fuzzyness)) {
-                $raw .= '~' . $this->fuzzyness;
+                if ($this->fuzzyness === self::FUZZY_AUTO) {
+                  $raw = '~';
```

Comment:
> We're considering using it in the Elasticsearch module for Drupal 8.

I guess that's why you indented with 2 space instead of 4 :) Just fix those 2 lines and I'll merge right after.

Many thanks!

Reply:
Ha! Totally. Coming right up.